cli/src/trusted_base_cli/commands/transfer.rs

```diff
@@ -55,12 +55,12 @@ impl TransferCommand {
 		let (mrenclave, shard) = get_identifiers(trusted_args);
 		let nonce = get_layer_two_nonce!(from, cli, trusted_args);
 		println!(
-			"send trusted call transfer from {} to {}: {}, nonce: {}, signing using mrenclave: {} and shard: {}",
-			from.public(),
-			to,
-			self.amount,
-			nonce, mrenclave.to_base58(), shard.0.to_base58()
-		);
+            "send trusted call transfer from {} to {}: {}, nonce: {}, signing using mrenclave: {} and shard: {}",
+            from.public(),
+            to,
+            self.amount,
+            nonce, mrenclave.to_base58(), shard.0.to_base58()
+        );
```

Comment:
fmt

Reply:
I did run fmt. so this just seems the new behavior?